### ISSUE TITLE
Admin & support team pages

### ISSUE THREAD

**Issue by lgebhardt** [brave-intl/publishers#763]
Provide a set of admin and support pages for the publishers site. The full feature set is still TBD. However to support #751 as a starting point the following is needed:

Models:
- Admin: Username and email as well as possibly permissions.

Admin only controllers (accessible only once logged in as an admin):
- UserController: 
   - Manage admin users
- OwnersController:
   - List, show, delete owners

Questions:
- do admins log in with a password, or an email token like publishers
- should 2FA be supported or required for admins
- do we expect to need different levels of admin permissions

**Comment by ayumi** [comment]
2FA should be required definitely.

All admin and support actions should be logged. The existing audit gem https://github.com/airblade/paper_trail can do that if you set the controller "#whodunnit".

This is a nice authorization gem:
https://github.com/CanCanCommunity/cancancan

**Comment by ayumi** [comment]
for an easy admin panel, administrate looks like it could be good:
https://github.com/thoughtbot/administrate/

**Comment by nvonpentz** [comment]
Closing in favor of #839
